practice-two/src/partials/components/pricing-card.html

```diff
@@ -1 +1,33 @@
-
+<div class="pricing-card {{class}}">
+  <div class="card-status">
+    <p>New</p>
```

Comment:
I think this is just a label on the card. So can we choose another tag element instead?